database/migrations/2019_02_04_214119_create_acceptances_table.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateAcceptancesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('acceptances', function (Blueprint $table) {
+            $table->uuid('id');
```

Comment:
Is there any benefit to considering attaching an acceptance just directly to a submission instead of to a conference and a talk revision? Again, braindead so I',m not sure.. just curious.

Reply:
I left it denormalized for the time being until I'm certain how we'll be querying for acceptances and because it's easier to drop columns than it is to add and backfill them. It may turn out that we don't need those relations and can drop them.

Reply:
👌